.github/workflows/deploy_docs.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy API Docs
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22"
```

Comment:
We should use a `.node-version` file and reference it in our workflows, but I didn't want to cause trouble in this PR